src/main/java/tc/oc/pgm/teams/TeamMatchModule.java

```diff
@@ -634,7 +634,7 @@ public boolean kickPlayerOffTeam(Team kickFrom, boolean forBalance) {
 
   @EventHandler(priority = EventPriority.MONITOR)
   public void onPartyChange(PlayerPartyChangeEvent event) {
-    if (event.getNewParty() instanceof Team) {
+    if (event.getOldParty() != null && event.getNewParty() != null) {
```

Comment:
Since this is `TeamMatchModule`, we will need to check if the `Party` is a `Team` and not a `Tribute`.

`event.getOldParty() instanceof Team && event.getNewParty() instanceof Team`

The reason this works is because `null instanceof <anything>` is always evaluated to be `false`.